app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -97,6 +94,25 @@ class NewConversationViewModel
         }
     }
 
+    private suspend fun tryGetAllContacts() {
+        innerSearchPeopleState = innerSearchPeopleState.copy(allKnownContacts = SearchResultState.InProgress)
+
```

Comment:
I think here you can remove the with context scope as the function is suspend and you call it inside view model scope 

and another suggestion

```suggestion
getAllContacts().let{
when(it){
is GetAllContactsResult.Failure -> {}
is GetAllContactsResult.Success -> {}
}
}
```

Reply:
I'll stick with val result = getAllContacts() as I was doing it in all other places already :P